api/DedicatedWorkerGlobalScope.json

```
@@ -52,31 +52,31 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/close",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
             },
             "chrome_android": {
               "version_added": true
```

OK, I did some testing and research and I can't find any evidence this was introduced later on Android

```suggestion
              "version_added": "18"
```